presto-native-execution/presto_cpp/main/tests/PrestoExchangeSourceTest.cpp

```diff
@@ -184,7 +184,7 @@ class Producer {
       future = std::move(f);
     }
 
-    std::move(future).get();
+    std::move(future).get(std::chrono::microseconds(10'000));
```

Comment:
This might cause time race? We want to avoid test timeout here if the code has bug? Thanks!

Reply:
@xiaoxmeng Not sure I understand. Would you clarify? If the future hasn't completed in 10s something is quite wrong, hence, it is fine to fail the test in that case.